taojieTan/assignments/treesDFS/lc113/lc113.py

```diff
@@ -0,0 +1,89 @@
+# Problem Statement #
+
+# Given a binary tree and a number ‘S’, find all paths from root-to-leaf such that the sum of all the node values of each path equals ‘S’.
```

Comment:
Great work again! Don't forget to include a note about the time and space complexity.